docs/configuration.md

```diff
@@ -1188,7 +1188,9 @@ Apart from these, the following properties are also available, and may be useful
   <td><code>spark.task.maxFailures</code></td>
   <td>4</td>
   <td>
-    Number of individual task failures before giving up on the job.
+    Number of failures of any particular task before giving up on the job. The same
+    task has to fail this number of attempts, it isn't affected by the number of failures
```

Comment:
for 2nd sentence maybe "Failures spread across different tasks will only cause the job to fail if one particular task has failed this number of times."?